modules/notes.js

```diff
@@ -55,7 +54,46 @@ const worker = async ({ channel, response_url, ts, text }) => {
     })
   }
 
-  if (channel !== DEV_CHANNEL_ID) {
+  let updates
+  try {
+    const { messages: thread } = await web.conversations.replies({ channel, ts })
+    // filter out any other message that could be conversation including parent note
+    updates = thread.filter(({ text }) => {
+      // get the first word of the block of text and check if it is an action
+      const match = text.match(/(?<action>[^/-]\S+)/)
+
+      // TODO: extract info when hugo is used to add updates as match is null, temp fix
+      if (match) {
+        const { action } = match.groups
+        return action.toLowerCase().match(/did|doing|issues|todo/)
+      }
+    })
+    /** updates = [
+   {
+    client_msg_id: '9e0d7786-423e-488b-8653-811f30c3dff9',
+    type: 'message',
+    text: 'Did - Locus:\n' +
+      '• FO/ML: new `/views` routes\n' +
+      '• FO/ML: expose content field\n' +
+      '• ML Writer: add validation for json fields\n' +
+      '• Atom/Onlia: split auto &amp; home conversions\n' +
+      '• Sales/Bell: translate deck into French\n' +
+      '• HH: calculate coverage metrics\n' +
+      'Doing:\n' +
+      '• ML: look into caching query results',
+    user: 'UU489FGKH',
+    ts: '1603810241.117100',
+    team: 'T1FDR4NG3',
+    blocks: [ [Object] ],
+    thread_ts: '1603807314.106800'
+  },
+  ...]
+  */
+  } catch(err) {
+    let text = err.message
+    if (text.includes('channel_not_found')) {
+      text = 'legion could not find channel. Make sure that legion is invited to this channel'
```

Comment:
maybe format it as `@legion` (or even better through legion's "user" ID) so it highlights/links to that bot user